docfx.json

```diff
@@ -186,9 +186,11 @@
         "docs/azure/**/**.md": "conceptual",
         "docs/core/tutorials/**.md": "tutorial",
         "docs/csharp/language-reference/**/**.md": "language-reference",
+        "docs/framework/**/*how-to*.md": "how-to",
```

Comment:
It's a really edge case, but what if "how" part of the file name was actually the end of another word. Wouldn't it be safer to remove the asterisk before "how"?

Reply:
>Is there a way to define an optional dash before "how"?

If you find such a way I'm happy to add it, but we don't need 100% accuracy, if there are exceptions they can be overridden in the doc itself.  This just establishes a default setting, which without it is "conceptual" for everything.